jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-test.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- job:
+    name: openshift-cluster-integreatly-test
+    display-name: 'Openshift Cluster Integreatly Test'
+    project-type: pipeline
+    parameters:
+      - string:
+          name: 'installerGitUrl'
+          default: 'https://github.com/integr8ly/installation.git'
+          description: '[REQUIRED] Integreatly installer Git URL'
+      - string:
+          name: 'installerGitBranch'
+          default: ''
```

Comment:
shouldn't we use "master" as default?

Reply:
I was considering master but I think this job will mostly be run for the suites branch: integreatly-next and integration-next, so I wasn't sure what default value to put in here.

Reply:
I don;t mind either way tbh, set a default of master if we think it's useful.